src/ExternalLink.tsx

```diff
@@ -1,15 +1,15 @@
-import React, { FunctionComponent, ReactNode } from 'react';
+import React from 'react';
 
 export interface ExternalLinkProps {
   href: string;
-  children?: ReactNode;
+  children?: React.ReactNode;
   [rest: string]: any;
 }
 
-const ExternalLink: FunctionComponent<ExternalLinkProps> = ({ href, children, ...rest }: ExternalLinkProps) => (
+const externalLink: React.FunctionComponent<ExternalLinkProps> = ({ href, children, ...rest }: ExternalLinkProps) => (
```

Comment:
Why do you change the first character to lowercase? It's a breaking change and has nothing to do with linting.

Also, react components usually start with uppercase by convention.

Reply:
Eslint was complaining about this.